qbft/spectest/tests/messages/marshal_justification.go

```diff
@@ -0,0 +1,41 @@
+package messages
+
+import (
+	"github.com/bloxapp/ssv-spec/qbft"
+	"github.com/bloxapp/ssv-spec/qbft/spectest/tests"
+	"github.com/bloxapp/ssv-spec/types"
+	"github.com/bloxapp/ssv-spec/types/testingutils"
+)
+
+// MarshalJustifications tests marshalling justifications
+func MarshalJustifications() *tests.MsgSpecTest {
+	ks := testingutils.Testing4SharesSet()
```

Comment:
if this test fails, what will the failure/error look like?

Reply:
likely
```
        	Error:      	Received unexpected error:
        	            	incorrect offset
```
or
```
        	Error:      	Received unexpected error:
        	            	incorrect size
```